stl/inc/ranges

```diff
@@ -1960,7 +1960,7 @@ namespace ranges {
             }
 
             _NODISCARD constexpr decltype(auto) operator*() const
-                _NOEXCEPT_IDL0(noexcept(_STD invoke(*_Parent->_Fun, *_Current))) /* strengthened */ {
+                _NOEXCEPT_IDL0(noexcept(_STD invoke(*_Parent->_Fun, *_Current))) {

```

Comment:
Previously, the Standard didn't mention `noexcept`, so we were free to partially strengthen it.

Now, the Standard mandates `noexcept(noexcept(invoke(*parent_->fun_, *current_)))`, so partially weakening it with `_NOEXCEPT_IDL0` is nonconformant.

I believe we need to say plain `noexcept` here. I'm not sure whether the function body needs to change. I also believe that the test needs to be updated:
https://github.com/microsoft/STL/blob/afe0800cd6bf0d163bf678ad4e1feb3c130d1ec5/tests/std/tests/P0896R4_views_transform/test.cpp#L489-L499